src/widgets/osd.cpp

```diff
@@ -183,6 +185,12 @@ void OSD::Paused() {
   }
 }
 
+void OSD::Resumed() {
```

Comment:
Just call it Playing and create a member variable that tracks whether it is paused or not.

Reply:
I don't really get what you mean with the member variable part, what would the use be?

Reply:
To prevent AlbumArtLoaded() done twice when Playing because it's done now in Resumed() too.